science/ecCodes/Portfile

```diff
@@ -3,10 +3,17 @@
 PortSystem          1.0
 PortGroup cmake     1.1
 PortGroup compilers 1.0
+PortGroup           legacysupport 1.1
+
+# Fix builds for macos <= 10.12.
+# Fix for undefined symbols for std::bad_optional_access::~bad_optional_access
+# Copied from https://trac.macports.org/ticket/69638
+legacysupport.newest_darwin_requires_legacy 16
+legacysupport.use_mp_libcxx yes
 
 name                ecCodes
 version             2.40.0
-revision            0
+revision            1
```

Comment:
no need to increase the "revision" when fixing a build failure.

Reply:
Okay, rev bump is cancelled.  I can not fix the capitalization using only the github web interface.  Please fix that on final merge.